index.js

```diff
@@ -11,11 +11,22 @@ var createSourceMapLocatorPreprocessor = function(args, logger, helper) {
     }
 
     function inlineMap(inlineData){
+
+      var charset = 'utf-8';
+      if (/^;charset=([^;]+);/.test(inlineData)) {
+        var matches = inlineData.match(/^;charset=([^;]+);/)
```

Comment:
Extracted it, but not to the top most scope, it's only used in this specific place, so I put it next to the charset var, hope that is okay.